bin/build

```diff
@@ -4,12 +4,15 @@
 # usage: ./bin/build
 set -ex
 
-. bin/build_utils
-
-CURRENT_DIR=$($(dirname $0)/abspath)
+CURRENT_DIR=$(cd $(dirname ${BASH_SOURCE[0]}); pwd)
```

Comment:
Ditto on comments about this in the other file

Reply:
addressed